### DISCUSSION TITLE
context deadline error

### DISCUSSION THREAD

**Discussion by alex60217101990** [olric-data/olric#133] [Technical Support]
[ERROR] Failed to check key count on partition: 222: context deadline exceeded => distribute.go:65
I get this error all the time when using Olric under load, both in local configuration and in replication mode.

**Comment by buraksezer** [comment]
Hi @alex60217101990 

There is an active discussion for this problem: https://github.com/buraksezer/olric/issues/130

Did you ever try to increase `PoolTimeout`? If you set zero to `PoolTimeout`, that error may change. Could you please try this? It's good to understand the nature of the problem.

In addition, what is the open connection count at the time of failure? You can inspect that via `olric-stats -n`
